src/reducers/change-fixture.js

```diff
@@ -0,0 +1,10 @@
+var CHANGE_FIXTURE = require('../actions/change-fixture.js').type;
+
+module.exports = function(state = {}, action) {
+  switch (action.type) {
+    case CHANGE_FIXTURE:
+      return action.data || {};
```

Comment:
Ah, I understand now what this action is supposed to do. By reading the action's payload I thought this reducer was meant to store the whole fixture. I suggest making it receive the entire fixture and only storing the redux data. The action is called `CHANGE_FIXTURE`, so it should obviously receive the fixture as the payload. I would name the reducer sto omething obvious too, like `entireStore` or just `store`.

Also, may I suggest doing

```js
const { store = {} } = action.fixture;

return store;
```

instead of that conditional?